packages/api/src/router/testSupport/personas.ts

```diff
@@ -0,0 +1,97 @@
+const GENERATIONS_PER_24H = parseInt(
```

Comment:
Is `/testSupport` essentially test helpers?

Reply:
Yep exactly. They're routes which are used by the e2e tests